prisma/schema.prisma

```diff
@@ -412,3 +428,10 @@ enum KycStatus {
   SUBMITTED
   SUCCESS
 }
+
+enum AllocationPool {
```

Comment:
We could make this lowercase to match the config here:
https://github.com/iron-fish/ironfish-api/blob/d8d32b077565cc5ac497b61da3d30bdd0b0a99a2/src/common/constants.ts#L69
But this will not match convention for enum. I am fine either way, we can do case conversion